src/PowerShellEditorServices/Services/PowerShellContext/PowerShellContextService.cs

```diff
@@ -213,27 +213,34 @@ public RunspaceDetails CurrentRunspace
                     hostUserInterface,
                     logger);
 
-            Runspace initialRunspace = PowerShellContextService.CreateRunspace(psHost, hostStartupInfo.LanguageMode);
+            if (hostStartupInfo.InitialSessionState.LanguageMode != PSLanguageMode.FullLanguage)
+            {
+                hostStartupInfo.InitialSessionState.ImportPSModule(hostStartupInfo.AdditionalModules.ToArray());
+                hostStartupInfo.InitialSessionState.ImportPSModule(new string[] { s_commandsModulePath });
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the array type; it is redundant.](https://app.codacy.com/manual/TylerLeonhardt/PowerShellEditorServices/pullRequest?prid=5462857)